platform/platform-configuration-listener/src/main/java/org/codice/ddf/configuration/store/ConfigurationFile.java

```diff
@@ -56,4 +62,49 @@ public Path getConfigFilePath() {
     }
 
     public abstract void createConfig() throws ConfigurationFileException;
+
```

Comment:
I'll need someone with more experience dealing with outputstream than myself.  I was under the impression other people were using the ```PersistanceStrategy.write```, but it looks like it was a previously unused method so it might not be fully baked.